src/storage.cc

```diff
@@ -115,6 +115,16 @@ Status Storage::SetColumnFamilyOption(const std::string &key, const std::string
   return Status::OK();
 }
 
+Status Storage::SetColumnFamilyOption(const std::string &cf_name, const std::string &key,
```

Comment:
how about
`Status Storage::SetColumnFamilyOption(const std::vector<std::string> &cf_names, const std::string &key,`
to handle multi CF setting options